### ISSUE TITLE
[Bug] Negative subquery used 

### ISSUE THREAD

**Issue by stevengoossensB** [elastic/detection-rules#3282]
**Describe the bug**
EQL constructed in the below manner is failing validation with error message **Negative subquery used** :
.
```
sequence by xxxxx with maxspan=15m
![subquery1]
[subquery2]
```

However, this should be legit EQL and we should be able to pass validation.

**Comment by Mikaayenson** [comment]
@stevengoossensB you are correct. It was added in 8.9.0. Can you provide an example TOML and/or confirm you're minstacked at 8.9?

**Comment by traut** [comment]
The validation is failing if the rule is min-stacked (`min_stack_version` field set) to the version lower than `8.9.0`. For the older versions the validation works correctly.
